aea/protocols/default/message.py

```diff
@@ -17,22 +17,27 @@
 #
 # ------------------------------------------------------------------------------
 
-"""This module contains the default message definition."""
+"""This module contains default's message definition."""
 
 from enum import Enum
-from typing import Any, Dict, cast
+from typing import Dict, Set, Tuple, cast
 
-from aea.configurations.base import PublicId
+from aea.configurations.base import ProtocolId
 from aea.protocols.base import Message
+from aea.protocols.default.custom_types import ErrorCode as CustomErrorCode
+
+DEFAULT_BODY_SIZE = 4
 
 
 class DefaultMessage(Message):
-    """The Default message class."""
+    """A protocol for exchanging any bytes message."""
+
+    protocol_id = ProtocolId("fetchai", "default", "0.1.0")
 
-    protocol_id = PublicId("fetchai", "default", "0.1.0")
+    ErrorCode = CustomErrorCode
```

Comment:
@5A11 all custom classes should by default be exposed here. Then they are accessible via `DefaultMessage.ErrorCode` which means only the `DefaultMessage` needs to be imported to access all its attributes and member classes.